crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -427,14 +427,8 @@ It can greatly improve performance (and readability) in such situations as it pr
             });
 
         if legend_visible.unwrap_or(true.into()).0 {
-            plot = plot.legend(
-                Legend::default().position(
-                    legend_corner
-                        .unwrap_or_default()
-                        .try_into()
-                        .unwrap_or(DEFAULT_LEGEND_CORNER),
-                ),
-            );
+            plot =
+                plot.legend(Legend::default().position(legend_corner.unwrap_or_default().into()));
```

Comment:
Legend::default() != `DEFAULT_LEGEND_CORNER` 
Probably it would be best if legend did _not_ have a Default impl since the default is supposed to be different for different views. (for the bar chart we want top right, for time series we want bottom right)

Also note that this changes makes the default legend inconsistent accross this file - in other places `DEFAULT_LEGEND_CORNER` is still used!